pymmonit/__init__.py

```diff
@@ -1,15 +1,25 @@
 # -*- coding: utf-8 -*-
+import netrc
```

Comment:
netrc, while great, is outside the scope of this module. There are millions of ways to load username and password, I don't think this module should be locked to netrc.